libs/langgraph/src/pregel/debug.ts

```diff
@@ -186,12 +188,24 @@ export function* mapDebugCheckpoint<
     return pyConfig;
   }
 
-  function getCurrentUTC() {
-    const now = new Date();
-    return new Date(now.getTime() - now.getTimezoneOffset() * 60 * 1000);
+  const parentNs = config.configurable?.checkpoint_ns;
+  const taskStates: Record<string, RunnableConfig | StateSnapshot> = {};
+
+  for (const task of tasks) {
+    if (!findSubgraphPregel(task.proc)) continue;
+
+    let taskNs = `${task.name as string}:${task.id}`;
+    if (parentNs) taskNs = `${parentNs}|${taskNs}`;
+
+    taskStates[task.id] = {
+      configurable: {
+        thread_id: config.configurable?.thread_id,
+        checkpoint_ns: taskNs,
+      },
+    };
   }
 
-  const ts = getCurrentUTC().toISOString();
+  const ts = new Date().toISOString();
```

Comment:
This is the only place which tries to return UTC date, whereas rest of the app uses `new Date()` instead. Opted to do the minimal change here, but can do the inverse as well. However not sure what would be the implications of doing so.

Reply:
This seems more natural